ArcGISRuntimeSDKQt_CppSamples/Maps/ReadGeoPackage/README.md

```diff
@@ -5,37 +5,45 @@ Add rasters and feature tables from a GeoPackage to a map.
 ![](screenshot.png)
 
 ## Use case
+
 The OGC GeoPackage specification defines an open standard for sharing raster and vector data. You may want to use GeoPackage files to support file-based sharing of geographic data.
 
 ## How to use the sample
+
 When the sample loads, the feature tables and rasters from the GeoPackage will be shown on the map.
 
 ## How it works
-1. Open the GeoPackage using GeoPackage::load
-2. Iterate through available rasters, exposed by GeoPackage::geoPackageRasters.
-3. For each raster, create a raster layer using the raster in the list, then add it to the map.
-4. Iterate through available feature tables, exposed by GeoPackage::geoPackageFeatureTables.
-5. For each feature table, create a feature layer using the feature table in the list, then add it to the map.
 
-## Relevant API
-- GeoPackage
-- GeoPackageRaster
-- GeoPackage::geoPackageRasters
-- GeoPackageFeatureTable
-- GeoPackage::geoPackageFeatureTables
+1. Open the GeoPackage using `GeoPackage::open(path)`.
```

Comment:
strange. We don't have an open method. You'd construct the GeoPackage with the path, then call load...

Reply:
@ldanzinger Seems so. I don't see an `open` method for Geopackage including from what was inherited.

I thought I caught this but I guess I didn't.